generated/_task_modules/channels/ai_channel.py

```diff
@@ -0,0 +1,202 @@
+                temp_size = 0
+            elif size_or_code > 0 and temp_size == 0:
+                # Buffer size obtained, use to retrieve data.
+                temp_size = size_or_code
+            else:
+                break
+
+        check_for_error(size_or_code)
+
+        return val.tolist()
+
+    @ai_bridge_poly_forward_coeff.setter
+    def ai_bridge_poly_forward_coeff(self, val):
+        val = numpy.float64(val)
+        cfunc = lib_importer.windll.DAQmxSetAIBridgePolyForwardCoeff
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        wrapped_ndpointer(dtype=numpy.float64,
+                        flags=('C','W'))]
+
+        error_code = cfunc(
+            self._handle, self._name, val)
+        check_for_error(error_code)
+
+    @ai_bridge_poly_forward_coeff.deleter
+    def ai_bridge_poly_forward_coeff(self):
+        cfunc = lib_importer.windll.DAQmxResetAIBridgePolyForwardCoeff
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str]
+
+        error_code = cfunc(
+            self._handle, self._name)
+        check_for_error(error_code)
+
+
+    @property
+    def ai_thrmcpl_cjc_chan(self):
+        """
+        Indicates the channel that acquires the temperature of the cold
+            junction if **ai_thrmcpl_cjc_src** is
+            **CJCSource1.SCANNABLE_CHANNEL**. If the channel is a
+            temperature channel, NI-DAQmx acquires the temperature in
+            the correct units. Other channel types, such as a resistance
+            channel with a custom sensor, must use a custom scale to
+            scale values to degrees Celsius.
+        """
+        cfunc = lib_importer.windll.DAQmxGetAIThrmcplCJCChan
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes_byte_str,
+                        ctypes.c_char_p, ctypes.c_uint]
+
+        temp_size = 0
+        while True:
+            val = ctypes.create_string_buffer(temp_size)
+
+            size_or_code = cfunc(
+                self._handle, self._name, val, temp_size)
+
+            if is_string_buffer_too_small(size_or_code):
+                # Buffer size must have changed between calls; check again.
+                temp_size = 0
+            elif size_or_code > 0 and temp_size == 0:
+                # Buffer size obtained, use to retrieve data.
+                temp_size = size_or_code
+            else:
+                break
+
+        check_for_error(size_or_code)
+
+        return str._factory(self._handle, val.value.decode('ascii'))
```

Comment:
This should return `Channel._factory(...)` not `str._factory(...)`.

![image](https://user-images.githubusercontent.com/17914399/219505940-60190669-4ed3-498e-b3d3-8364218f8c62.png)

Reply:
Updated the getter template to return `object_type` instead of `python_data_type`, if the attribute has `object_type`